airbyte-integrations/connectors/destination-iceberg-v2/gradle.properties

```diff
@@ -1 +1 @@
-testExecutionConcurrency=-1
+testExecutionConcurrency=1
```

Comment:
nit: add a comment explaining why (... I don't actually remember why we need this)

Reply:
I actually think that this can be reverted now